### ISSUE TITLE
Spanish Support

### ISSUE THREAD

**Issue by cgomezmendez** [teamforus/me-android#45]
There is  437 million people that speaks spanish as native language, would be nice to have app translated in spanish.

**Comment by maxvisser** [comment]
Yes. The use case is now a small municipality in the Netherlands (80 users) but if we can add more use cases. It will get more usefull.:smile:
